### ISSUE TITLE
[Entrega 3] Problema al hacer git pull desde la carpeta Sites

### ISSUE THREAD

**Issue by Crisax234** [IIC2413/Syllabus-2022-1#333]
Hola, cuando hago git pull desde la carpeta sites del servidor para subir mis archivos, me aparece este error:
![image](https://user-images.githubusercontent.com/88630837/175448908-92cef7eb-49e8-46c1-bc16-db13a38d7f60.png)
Entiendo que sale que no tengo memoria, pero no se a que se refiere, como se puede solucionar?, desde ya muchas gracias.

**Comment by Bizonte28** [comment]
A mi me sale un error similar al intentar subir un archivo desde filezilla:

Error:	error while writing: received failure with description 'Failure'
Error:	Transferencia fallida

Según lo que encontré en internet es porque la memoria del servidor está llena.

**Comment by arpincheira** [comment]
Efectivamente, estamos viendo como solucionar eso, especialmente porque hay grupos que ocupan una gran cantidad de megabytes dentro de sus carpetas por uso indebido del servidor a partir de aplicaciones como la extensión de ssh de vs-code.
